src/lib.rs

```diff
@@ -108,6 +111,8 @@ impl Router {
     /// modules. It will call the `_routes()` method on each module. If caching is
     /// enabled, it will also clear and recreate the module cache.
     fn reload(&self) -> anyhow::Result<()> {
+        log::info!("Nothing reloaded.");
```

Comment:
Super nit (especially because I know the follow up PR is gonna fix it): Should probably be a `warn!`